mm2src/coins/tendermint/tendermint_coin.rs

```diff
@@ -85,7 +88,7 @@ const ABCI_REQUEST_PROVE: bool = false;
 /// 0.25 is good average gas price on atom and iris
 const DEFAULT_GAS_PRICE: f64 = 0.25;
 pub(super) const TIMEOUT_HEIGHT_DELTA: u64 = 100;
-pub const GAS_LIMIT_DEFAULT: u64 = 100_000;
+pub const GAS_LIMIT_DEFAULT: u64 = 150_000;
```

Comment:
Reason: Cross chain ibc transfers requires a little higher gas than our default limit.

Reply:
No, they don't pay more fees. It's the limit that fails the transaction if used gass is more than the gas limit. I created `IBC_GAS_LIMIT_DEFAULT`, and rollback `GAS_LIMIT_DEFAULT` to it's old value.